src/Microsoft.AspNetCore.Razor.Language/Syntax/SyntaxKind.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNetCore.Razor.Language
+{
+    internal enum SyntaxKind : byte
+    {
+        Unknown,
+        List,
+        Whitespace,
+        NewLine,
+
+        // HTML
+        HtmlText,
+        HtmlDocument,
+        HtmlDeclaration,
+        HtmlTextLiteralToken,
+        OpenAngle,
+        Bang,
+        ForwardSlash,
+        QuestionMark,
+        DoubleHyphen,
+        LeftBracket,
+        CloseAngle,
+        RightBracket,
+        Equals,
+        DoubleQuote,
+        SingleQuote,
+        Transition,
+        Colon,
+    }
```

Comment:
Is this taken from an existing enum somwhre in Razor?

Reply:
Yup. https://github.com/aspnet/Razor/blob/master/src/Microsoft.AspNetCore.Razor.Language/Legacy/HtmlTokenType.cs